packages/cli/src/cmds/m365/m365.ts

```diff
@@ -59,96 +57,138 @@ async function getTokenAndUpn(): Promise<[string, string]> {
   }
 }
 
-async function sideLoading(baseUrl: string, token: string, manifestPath: string): Promise<void> {
-  const data = await fs.readFile(manifestPath);
-
-  const instance = axios.create({
-    baseURL: baseUrl,
-    timeout: 30000,
-  });
-  instance.defaults.headers.common["Authorization"] = `Bearer ${token}`;
-
-  try {
-    const content = new FormData();
-    content.append("package", data);
-    CLILogProvider.necessaryLog(LogLevel.Info, "Uploading package ...");
-    const uploadResponse = await instance.post(
-      "/dev/v1/users/packages",
-      content.getBuffer(),
-      content.getHeaders()
-    );
+class M365Sideloading extends YargsCommand {
+  public readonly commandHead = "sideloading";
```

Comment:
Have the command names already confirmed by PM?

Reply:
The `m365` command is for office-addins integration only. It has *preview* warning and is hidden from `--help`.
The names are confirmed by office-addins team. Actually they already used a private build with these names, and this PR is make these commands public for easier integration.